internal-packages/eslint-config-next-on-pages/index.js

```diff
@@ -8,7 +8,7 @@
 	"parser": "@typescript-eslint/parser",
 	"parserOptions": {
 		"sourceType": "module",
-		"project": "./tsconfig.json"
+		"project": "tsconfig.json"
```

Comment:
Why this change and what does it mean to have no ./?

Reply:
I'll have to have a look, I can't remember why I made the change honestly 😓 

I think it's either that it wasn't working as intended or that `./` doesn't make any difference 🤔

Reply:
Yeah I just checked and it doesn't seem to make any difference, the value there is a glob path: https://typescript-eslint.io/linting/typed-linting#specifying-tsconfigs so I think that unless it starts with `/` it is considered relative

I can add back the `./` if you find that more explicit/clear

I could also set this to `true` and it should pick up the correct `tsconfig` anyways if you'd prefer that 🙂 

___

Actually I did set it to `true` since it seems the proper solution given that the package is shared and doesn't have a `tsconfig` itself. So with `true` it feels to me clearer that we just delegate the `tsconfig` lookup to the consuming package, what do you think?